lib/shared/addon/components/container-shell/component.js

```diff
@@ -139,6 +140,23 @@ export default Component.extend(ThrottledResize, {
 
     set(this, 'socket', socket);
 
+    socket.onclose = (err = {}) => {
+      const term = get(this, 'term')
+
+      if (err.code === 1006) {
+        get(this, 'growl').fromError(`WebSocket connection to '${ socket.url }', failed: Error during WebSocket handshake: Unexpected response code: 404`)
```

Comment:
Localize error

Reply:
This message makes no sense, websocket code 1006 ([connection closed abnormally](https://tools.ietf.org/html/rfc6455#section-7.4.1) has nothing to do with http status 404.